doc/source/api/visualization/graphics_windows_manager.rst

```diff
@@ -6,40 +6,39 @@ The ``GraphicsWindowsManager`` class provides for managing and directly interact
 with PyVista windows. By registering these methods with ``EventsManager``, you
 can update graphics during run time and create animations.
 
-The following example updates ``window-1`` during solution initialization and
-whenever data is read. During the calculation, it also updates ``window-1`` at
+The following example updates ``p_cont`` during solution initialization and
+whenever data is read. During the calculation, it also updates ``p_cont`` at
 the end of every time step and creates an animation.
 
 .. note::
   The animation is saved when the window is closed.
 
 .. code-block:: python
 
-    from ansys.fluent.visualization import Graphics
-    from ansys.fluent.visualization.graphics import graphics_windows_manager
-    
-    graphics_session = Graphics(session)
+    from ansys.fluent.visualization import Contour, GraphicsWindow
     
     #Create contour.
-    contour1 = graphics_session.Contours["contour-1"]
-    contour1.field = "velocity-magnitude"
-    contour1.surfaces = ['symmetry']
+    contour1 = Contour(
+        solver=session, field="velocity-magnitude", surfaces=["symmetry"]
+    )
     
-    #Display contour on window-1.
-    contour1.display("window-1")
+    #Display contour.
+    p_cont = GraphicsWindow()
+    p_cont.add_graphics(contour1)
+    p_cont.show()
     
-    #Create callback that refreshes window-1.    
-    def auto_refresh_contour(session_id, event_info):    
-        graphics_windows_manager.refresh_windows(session_id, ["window-1"])
-           
+    #Create callback that refreshes 'p_cont'.
+    def auto_refresh_contour(session, event_info):
+        # 'p_cont' uses 'graphics_windows_manager' in its implementation
+        p_cont.refresh_windows(session.id)
+
     #Register this callback with server events.    
-    cb_init_id = session.events_manager.register_callback('InitializedEvent', auto_refresh_contour)
-    cb_data_read_id = session.events_manager.register_callback('DataReadEvent', auto_refresh_contour)
-    cb_time_step_ended_id = session.events_manager.register_callback('TimestepEndedEvent', auto_refresh_contour)         
+    cb_init_id = session.events.register_callback('InitializedEvent', auto_refresh_contour)
+    cb_data_read_id = session.events.register_callback('DataReadEvent', auto_refresh_contour)
+    cb_time_step_ended_id = session.events.register_callback('TimestepEndedEvent', auto_refresh_contour)
 
-    #Create animation for window-1. 
-    graphics_windows_manager.animate_windows(session.id, ["window-1"])
-    
+    #Create animation.
+    p_cont.animate_windows(session.id)
```

Comment:
`p_cont` was instantiated as a `GraphicsWindow` so I'm having difficulty relating it to `animate_windows`. I note that in the old example, we had `graphics_windows_manager.animate_windows(session.id, ["window-1"])` and this haas simply been carried over. Logically, it should be simply `p_cont.animate(session.id)`.